README.md

```diff
@@ -140,16 +193,25 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+RESTful uses different HTTP methods to represent
+different actions. In short, Emperor, it gives you more control. If you'd like to move
+one or many soldiers at a time,  or find an individual soldier's id, you can do so.
 ```
 Your routes:
 ```
 The ancestors have provided an example of one route; you do the other six!
 
 GET '/warriors/:id'
-  * This is the show route, which finds a warrior by ID, and displays information about that warrior.
+  * This is the show route, which finds a warrior by ID, and displays information about
+  that warrior.
+
+1. '/warriors/'
+2. '/warriors/news'
+3. '/warriors'
+4. '/warriors/:id/edit'
+5. 'warriros/:id'
+6.  'warriros/:id'
```

Comment:
You got the routes down for the most part (although look at your syntax/spelling on 2, 5 and 6).

You need to match them, however, with the proper HTTP request (e.g., GET, POSTS).